test/transaction_test_hosted.dart

```diff
@@ -1,18 +1,19 @@
 import 'package:flutter/foundation.dart';
+import 'package:flutter_app/api_calls/cosmos_api.dart';
 import 'package:flutter_app/api_calls/faucet_api.dart';
-import 'package:flutter_app/api_calls/wallet_api.dart';
 import 'package:flutter_app/global.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
```

Comment:
lets stick with a naming convetion that all test files are ending with `*_test.dart` suffix. this will be easier to distinguish the files while reviewing. here: `hosted_transaction_test.dart`

Reply:
✅